### ISSUE TITLE
Startup crash (0x1b9538964)

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero-ios#590]
https://forums.zotero.org/discussion/101381/crash-log-104752001

```
Exception Type:  SIGABRT
Exception Codes: #0 at 0x1b9538964
Crashed Thread:  0

Thread 0 Crashed:
0   libsystem_kernel.dylib              0x00000001b9538964 0x1b9531000 + 31076
1   libsystem_c.dylib                   0x000000018e033f50 0x18e014000 + 130896
2   Zotero                              0x0000000103a6f2c4 0x102fc8000 + 11170500
3   Zotero                              0x0000000103a6f604 0x102fc8000 + 11171332
4   Zotero                              0x0000000103a6f380 0x102fc8000 + 11170688
5   Zotero                              0x00000001038192f4 0x102fc8000 + 8721140
6   Zotero                              0x0000000103828990 0x102fc8000 + 8784272
7   Zotero                              0x0000000103828c80 0x102fc8000 + 8785024
8   Zotero                              0x0000000103a2a880 0x102fc8000 + 10889344
9   Zotero                              0x0000000103841dbc 0x102fc8000 + 8887740
10  Zotero                              0x0000000103841a50 0x102fc8000 + 8886864
11  Zotero                              0x0000000103a4740c 0x102fc8000 + 11006988
12  Zotero                              0x000000010382b8bc 0x102fc8000 + 8796348
13  Zotero                              0x000000010382dc10 0x102fc8000 + 8805392
14  Zotero                              0x0000000103831920 0x102fc8000 + 8821024
15  Zotero                              0x00000001038bcab4 0x102fc8000 + 9390772
16  Zotero                              0x00000001038bd9a0 0x102fc8000 + 9394592
17  Zotero                              0x00000001038bd830 0x102fc8000 + 9394224
18  Zotero                              0x000000010391b90c 0x102fc8000 + 9779468
19  Zotero                              0x000000010372e5d0 0x102fc8000 + 7759312
20  Zotero                              0x00000001037c633c 0x102fc8000 + 8381244
21  Zotero                              0x00000001037c684c 0x102fc8000 + 8382540
22  Zotero                              0x00000001031fca7c 0x102fc8000 + 2312828
23  Zotero                              0x000000010319ea4c 0x102fc8000 + 1927756
24  Zotero                              0x000000010319dac0 0x102fc8000 + 1923776
25  Zotero                              0x000000010319c53c 0x102fc8000 + 1918268
26  Zotero                              0x000000010319c0cc 0x102fc8000 + 1917132
27  Zotero                              0x000000010313f444 0x102fc8000 + 1537092
28  Zotero                              0x00000001032f9cf8 0x102fc8000 + 3349752
29  UIKitCore                           0x0000000185758f34 0x185603000 + 1400628
30  UIKitCore                           0x0000000185d31c0c 0x185603000 + 7531532
31  UIKitCore                           0x00000001859ac81c 0x185603000 + 3840028
32  UIKitCore                           0x0000000185ef3f40 0x185603000 + 9375552
33  UIKitCore                           0x0000000185844918 0x185603000 + 2365720
34  UIKitCore                           0x00000001858c8584 0x185603000 + 2905476
35  UIKitCore                           0x0000000185793d4c 0x185603000 + 1641804
36  UIKitCore                           0x00000001857a8598 0x185603000 + 1725848
37  UIKitCore                           0x00000001857a8700 0x185603000 + 1726208
38  QuartzCore                          0x0000000186f7601c 0x186ead000 + 823324
39  libdispatch.dylib                   0x0000000182e68660 0x182e64000 + 18016
40  libdispatch.dylib                   0x0000000182e76b60 0x182e64000 + 76640
41  CoreFoundation                      0x00000001831aecd4 0x18315d000 + 335060
42  CoreFoundation                      0x0000000183168eac 0x18315d000 + 48812
43  CoreFoundation                      0x000000018317c3b8 0x18315d000 + 127928
44  GraphicsServices                    0x000000019eb0c38c 0x19eb0b000 + 5004
45  UIKitCore                           0x0000000185b1c6a8 0x185603000 + 5346984
46  UIKitCore                           0x000000018589b7f4 0x185603000 + 2721780
47  Zotero                              0x0000000102fcdc3c 0x102fc8000 + 23612
48  ???                                 0x0000000104415a24 0x0 + 0
```

**Comment by michalrentka** [comment]
Related to #555, hopefully solved in https://github.com/realm/realm-core/issues/5937, we're waiting for a release with given fix.

**Comment by michalrentka** [comment]
Should be fixed in 66ac152a28fb5d5f294e5fdc2bd65605a258b365, affected users have to re-install unfortunately, but we shouldn't see new cases of this.
